src/App.tsx

```diff
@@ -1,14 +1,73 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+
+type Params = {
+  query: string;
+  filter: string;
+};
+
+function filterTodos(list: Todo[], params: Params) {
+  const { query, filter } = params;
+  let result: Todo[] = [];
+
+  if (filter) {
+    switch (filter) {
+      case 'all':
```

Comment:
The filter value here is `'all'`, which is consistent with the initialization in the `useState` hook. Ensure that all filter values are consistently used in lowercase throughout the application.